src/components/productdisplayitem.main.less

```diff
@@ -28,19 +28,28 @@
 
   .social-network-sharing {
     margin-top: 15px;
+    .st-btn {
+      min-width: 50px !important;
+      >span {
+        display: none !important;
+      }
+    }
   }
 
   .itemdetail-assets {
-    width: 380px;
+    width: 50%;
+    position: sticky;
```

Comment:
With this current level, the image will overlap with the subsequent divs (like the recommended products). 
![image](https://user-images.githubusercontent.com/39800563/51134026-6f6f1f00-1804-11e9-9612-80bbdd510f5b.png)

Need to sticky it up to a certain point:
![image](https://user-images.githubusercontent.com/39800563/51134056-7e55d180-1804-11e9-94a6-ce364803bcf1.png)

You can use this product locally to test with:
https://reference.elasticpath.com/itemdetail/CLOTHES_BUNDLE